.github/workflows/scripts/run-shellcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --rm -v "$PWD:/mnt" koalaman/shellcheck:v0.9.0 -P ./bin/ -x ./scripts/*.sh
```

Comment:
@CalvinRodo What does the -P parameter usually targets? and do I have the right positional argument to target the location of the scripts to check?